calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -144,20 +150,18 @@ protected function generateFiles($arrFeed)
 					continue;
 				}
 
+				$objParent = $this->getPageWithDetails($jumpTo);
+
+				// A jumpTo page is set but does no longer exist (see #5781)
+				if ($objParent === null)
+				{
+					continue;
+				}
+
 				// Get the jumpTo URL
 				if (!isset($arrUrls[$jumpTo]))
 				{
-					$objParent = PageModel::findWithDetails($jumpTo);
-
-					// A jumpTo page is set but does no longer exist (see #5781)
-					if ($objParent === null)
-					{
-						$arrUrls[$jumpTo] = false;
-					}
-					else
-					{
-						$arrUrls[$jumpTo] = $objParent->getAbsoluteUrl(Config::get('useAutoItem') ? '/%s' : '/events/%s');
-					}
+					$arrUrls[$jumpTo] = $objParent->getAbsoluteUrl(Config::get('useAutoItem') ? '/%s' : '/events/%s');
 				}
 
 				// Skip the event if it requires a jumpTo URL but there is none
```

Comment:
`$arrUrls[$jumpTo] === false` can no longer happen, right?

Reply:
True, we can probably remove this altogether:

https://github.com/contao/contao/blob/8b7007185f3a82b637e42860b637d82641cb8a38/calendar-bundle/src/Resources/contao/classes/Calendar.php#L167-L171

Reply:
Done.